app/data/formPages/types.ts

```diff
@@ -1,5 +1,6 @@
 export interface SummaryFormProps<TProjectRevision> {
   projectRevision: TProjectRevision;
+  query?: any;
```

Comment:
This is a little redundant, since the project revision will be on the query when we use that. There might be a better way to organize those summary form props to make the query/fragments a little cleaner.

This PR is already large and this works so we can merge this, but I'd like to put in a tech debt card to refactor this

Reply:
https://github.com/bcgov/cas-cif/issues/1584